pkg/infra/iac2/templates_compiler.go

```diff
@@ -664,6 +664,17 @@ func (tc TemplatesCompiler) handleIaCValue(v core.IaCValue, appliedOutputs *[]Ap
 			tc.getVarName(region), tc.getVarName(v.Resource)), nil
 	case resources.NODE_GROUP_NAME_IAC_VALUE:
 		return fmt.Sprintf(`%s.nodeGroupName`, tc.getVarName(resource)), nil
+	case resources.ALB_CONTROLLER_READINESS_GATE_IAC_VALUE:
+		return `
+(obj: any, opts: pulumi.CustomResourceOptions) => {
+	if (obj.kind === "MutatingWebhookConfiguration") {
+		obj.webhooks.filter(w => w.name === "mpod.elbv2.k8s.aws").forEach((webhook: any) => {
+                webhook.namespaceSelector = undefined
+                webhook.objectSelector.matchLabels = {"elbv2.k8s.aws/pod-readiness-gate-inject": "enabled"}
+            }
+		)
+	}	
+}`, nil
```

Comment:
i dont think this is needed.
look for enablePodReadinessGateInject in
https://github.com/kubernetes-sigs/aws-load-balancer-controller/tree/main/helm/aws-load-balancer-controller

 i think that covers it

Reply:
Thanks for pointing me in the direction of the ALB Controller chart's values. I was looking at an older version that didn't template the selectors we need to modify for this change.

`enablePodReadinessGateInject` isn't required unless you want to disable pod readiness gate injection.

I've reworked this PR to make use of the chart's `objectSelector.matchLabels` and `webhookNamespaceSelectors` values to apply the same changes originally included in the transformation.